packages/api/src/schema/api-schema.yaml

```diff
@@ -2929,7 +2958,7 @@ paths:
           content:
             application/json:
               schema:
-                type: array
+                type: object
```

Comment:
Hmm this seems wrong. `type: object` wouldn't have an `items:` prop below. Maybe missing changing `items { $ref: ... }` to just `$ref` on line below?

Reply:
I guess, we could just replace it with this:

```
application/json:
              schema:
                $ref: "#/components/schemas/multistream-target"
```

as the API returns object not array

Reply:
Yeah that makes sense!